Base/usr/share/man/man7/boot_device_addressing.md

```diff
@@ -0,0 +1,61 @@
+## Name
+
+Boot Device Addressing - addressing the correct boot device to use.
+
+## Description
+
+Serenity's kernel can select boot device at boot time, based on the `root` boot parameter.
+This functionality is used to control which boot device is selected to be used for all further boot process operations.
+
+## Options
+
+The kernel `root` boot parameter take the form of **`root={value}`**, where the **`={value}`**
+trailer can be set to specific prefixes to indicate the boot device preference.
+
+### Addressing options
+
+The user can choose to use addressing based on synthetic unix concepts:
+
+```
+block0,0 [blockMAJOR:MINOR]
+```
+
+This is especially useful in static hardware setups, so the user can choose to use
+either a raw `StorageDevice` or partition block device.
+
+However, when there's knowledge of the hardware arrangement of raw `StorageDevice`s,
+it could be valuable to use addressing based on hardware-relative interface-specific "location"
+to address raw `StorageDevice`s:
+
+```
+ata0,0,0 [First ATA controller, ATA first primary channel, master device]
+nvme0,0 [First NVMe Controller, First NVMe Namespace]
+ramdisk0 [First Ramdisk]
+```
+
+When the logical arrangement is known, using (absolute) LUNs is the easiest option as it doesn't rely on
+using unix device numbers or hardware-relative location:
+
+```
+lun0:0:0 - first device on the first channel of the first controller to be enumerated
```

Comment:
why does this use `:` but all of the above use `,`?

Reply:
Because it uses the same naming convention like in the SysFS which is a special case. Other addressing methods are not exposed in the SysFS. I do agree that it is a bit confusing, so I'll choose `:` instead of `,`.